tr/cs-221-states-models.md

```diff
@@ -0,0 +1,980 @@
+
+&#10230; Not: Eğer γ<1 ya da Markov karar süreci (Markov Decision Process - MDP) asiklik (çevrimsiz) olursa, o zaman değer tekrarı algoritmasının doğru cevaba yakınsayacağı garanti edilir.
+
+<br>
+
+
+**80. When unknown transitions and rewards**
+
+&#10230; Bilinmeyen geçişler ve ödüller
+
+<br>
+
+
+**81. Now, let's assume that the transition probabilities and the rewards are unknown.**
+
+&#10230; Şimdi, geçiş olasılıklarının ve ödüllerin bilinmediğini varsayalım.
+
+<br>
+
+
+**82. Model-based Monte Carlo ― The model-based Monte Carlo method aims at estimating T(s,a,s′) and Reward(s,a,s′) using Monte Carlo simulation with: **
+
+&#10230; Model-temelli Monte Carlo ― Model-temelli Monte Carlo yöntemi, T(s,a,s′) ve Reward(s,a,s′) işlevlerini Monte Carlo benzetimi kullanarak aşağıdaki formüllere uygun bir şekilde tahmin etmeyi amaçlar:
+
+<br>
+
+
+**83. [# times (s,a,s′) occurs, and]**
+
+&#10230; [# (s,a,s′) gerçekleşme sayısı, ve]
+
+<br>
+
+
+**84. These estimations will be then used to deduce Q-values, including Qπ and Qopt.**
+
+&#10230; Bu tahminler daha sonra Qπ ve Qopt'yi içeren Q-değerleri çıkarımı için kullanılacaktır.
+
+<br>
+
+
+**85. Remark: model-based Monte Carlo is said to be off-policy, because the estimation does not depend on the exact policy.**
+
+&#10230; Not: model-tabanlı Monte Carlo'nun politika dışı olduğu söyleniyor, çünkü tahmin kesin politikaya bağlı değildir.
+
+<br>
+
+
+**86. Model-free Monte Carlo ― The model-free Monte Carlo method aims at directly estimating Qπ, as follows:**
+
+&#10230; Model içermeyen Monte Carlo ― Model içermeyen Monte Carlo yöntemi aşağıdaki şekilde doğrudan Qπ'yi tahmin etmeyi amaçlar:
+
+<br>
+
+
+**87. Qπ(s,a)=average of ut where st−1=s,at=a**
+
+&#10230; Qπ(s,a)= ortalama ut , st−1=s ve at=a olduğunda
+
+<br>
+
+
+**88. where ut denotes the utility starting at step t of a given episode.**
+
+&#10230; ut belirli bir bölümün t anında başlayan faydayı ifade etmektedir.
+
+<br>
+
+
+**89. Remark: model-free Monte Carlo is said to be on-policy, because the estimated value is dependent on the policy π used to generate the data.**
+
+&#10230; Not: model içermeyen Monte Carlo'nun politikaya dahil olduğu söyleniyor, çünkü tahmini değer veriyi üretmek için kullanılan π politikasına bağlıdır.
+
+<br>
+
+
+**90. Equivalent formulation - By introducing the constant η=11+(#updates to (s,a)) and for each (s,a,u) of the training set, the update rule of model-free Monte Carlo has a convex combination formulation:**
+
+&#10230; Eşdeğer formülasyon - Sabit tanımı η=11+(#güncelleme sayısı (s,a) ) ve eğitim kümesinin her bir (s,a,u) üçlemesi için, model içermeyen Monte Carlo'nun güncelleme kuralı dışbükey bir kombinasyon formülasyonuna sahiptir:
+
+<br>
+
+
+**91. as well as a stochastic gradient formulation:**
+
+&#10230; olasılıksal bayır formülasyonu yanında:
```

Comment:
stochastic gradient için "stokastik bayır (stochastic gradient)" kullanmıştık öncekilerde

Reply:
stokastik aslında olasılıksal olduğu için Türkçe olsun diye yazmıştım.